src/EnergyPlus/CurveManager.cc

```diff
@@ -2547,6 +2554,18 @@ namespace CurveManager {
 				}
 			}
 
+			IndVarSwitch = false;
+			if ( SameString( Alphas( 4 ), "WAVELENGTH" ) && SameString( Alphas( 5 ), "ANGLE" ) ) {
+				IndVarSwitch = true;
```

Comment:
No problem, I can force variable types to specify X1 as angle and X2 as wavelength.